### PR TITLE
TST/REF: collect tests by method

### PR DESCRIPTION
**Pull request by jbrockmendel**

### CODE REVIEW

File: pandas/tests/frame/test_nonunique_indexes.py [hunk 1]
```diff
@@ -400,29 +400,6 @@ def check(result, expected=None):
         result = z.loc[["a", "c", "a"]]
         check(result, expected)
 
-    def test_column_dups_indexing2(self):
```
Comment (jreback) [inline pandas/tests/frame/test_nonunique_indexes.py:403]: where did this go?

File: pandas/tests/series/test_timeseries.py [hunk 2]
```diff
@@ -1,26 +1,10 @@
 import numpy as np
 
-import pandas as pd
 from pandas import DataFrame, Series, date_range, timedelta_range
 import pandas._testing as tm
 
 
 class TestTimeSeries:
-    def test_contiguous_boolean_preserve_freq(self):
```
Comment (jreback) [inline pandas/tests/series/test_timeseries.py:9]: where did this go?

**Review by jreback — CHANGES_REQUESTED** [review]

File: pandas/tests/series/test_timeseries.py (hunk 2 above)
Comment (jbrockmendel) [reply pandas/tests/series/test_timeseries.py:9]: test_arithmetic_with_duplicate_index

Comment (jbrockmendel) [reply pandas/tests/series/test_timeseries.py:9]: woops, meant test_getitem_boolean_contiguous_preserve_freq

File: pandas/tests/frame/test_nonunique_indexes.py (hunk 1 above)
Comment (jbrockmendel) [reply pandas/tests/frame/test_nonunique_indexes.py:403]: test_arithmetic_with_duplicate_index

**Review by jreback — APPROVED** [review]

**Comment by jbrockmendel** [comment]
One group of tests im having trouble finding an "obvious" home for: ones directly setting .columns/.index/.name attrs.  Thoughts?

**Pull request merged**